### DISCUSSION TITLE
Will the `0.5-compat` branch be updated for non breaking changes?

### DISCUSSION THREAD

**Discussion by David-Else** [nvim-treesitter/nvim-treesitter#1701] [Q&A] [answered]
Have any/all the updates since the `0.5-compat` branch was created been breaking for Neovim 0.5? If not, will they be backported to `0.5-compat` in the near future? Cheers.

**Comment by theHamsta** [comment, marked as answer]
No, there have been no incompatible changes until now. We could think about setting the 0.5-branch to master
